templates/base/api/base.ts.hbs

```diff
@@ -1,8 +1,8 @@
 import axios from "axios";
 import * as T from "./types";
 
-export const COORDINATOR_HOST = process.env.COORDINATOR_HOST!;
-export const MATCHMAKER_HOST = process.env.MATCHMAKER_HOST!;
+export const COORDINATOR_HOST = process.env.COORDINATOR_HOST ?? "coordinator.hathora.dev";
```

Comment:
can we handle APP_SECRET and DATA_DIR in the same way? for consistency sake

I know they're not runtime args right now, but would prefer making them runtime args

Reply:
yeah can do that as a separate change

Reply:
and can we get rid of STORE_REGION?